static/app/views/organizationGroupDetails/grouping/newIssue.tsx

```diff
@@ -1,104 +1,65 @@
+import {Fragment} from 'react';
 import styled from '@emotion/styled';
 
 import EventOrGroupHeader from 'app/components/eventOrGroupHeader';
-import ProjectBadge from 'app/components/idBadge/projectBadge';
-import {PanelItem} from 'app/components/panels';
-import ShortId from 'app/components/shortId';
 import TimeSince from 'app/components/timeSince';
 import {IconClock} from 'app/icons';
-import {tn} from 'app/locale';
+import {t} from 'app/locale';
 import space from 'app/styles/space';
-import {Organization, Project} from 'app/types';
+import {Organization} from 'app/types';
 import {Event} from 'app/types/event';
 
 type Props = {
   sampleEvent: Event;
   eventCount: number;
   organization: Organization;
-  project?: Project;
 };
 
-function NewIssue({sampleEvent, eventCount, organization, project}: Props) {
+function NewIssue({sampleEvent, eventCount, organization}: Props) {
   return (
-    <StyledPanelItem>
-      <div>
+    <Fragment>
+      <EventDetails>
         <EventOrGroupHeader
           data={sampleEvent}
           organization={organization}
           hideIcons
           hideLevel
         />
-        <Details>
-          {project && (
-            <GroupShortId
-              shortId={project.slug}
-              avatar={
-                project && <ProjectBadge project={project} avatarSize={14} hideName />
-              }
-              onClick={e => {
-                // prevent the clicks from propagating so that the short id can be selected
-                e.stopPropagation();
-              }}
-            />
-          )}
+        <ExtraInfo>
           <TimeWrapper>
-            <IconClock size="xs" />
-            <TimeSince date={sampleEvent.dateCreated} />
+            <IconClock size="11px" color="gray300" />
```

Comment:
should this be a color alias to get proper contrast in dark mode?

Reply:
We usually use `gray300` in an `iconClock` and it looks good also in dark mode  😉